kalliope/core/OrderAnalyser.py

```diff
@@ -44,7 +44,7 @@ def get_matching_synapse(cls, order, brain=None):
 
         # We use a namedtuple to associate the synapse and the signal of the synapse
         synapse_order_tuple = collections.namedtuple('tuple_synapse_matchingOrder',
```

Comment:
We probably need to get rid of this tuple at some point ... it will become messy on the long run